apps/ima/views/life-or-liberty-threatened.html

```diff
@@ -0,0 +1,15 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>Your life or liberty must have been threatened because of your:</p>
+    <ul class="list list-bullet">
```

Comment:
You will need to indent the list by margin-right : 10px

Reply:
Added indentation as suggested